data/hai/hai news.txt

```diff
@@ -91,6 +99,74 @@ news "cute story in Hai space"
 			"I saw the Hai-human fusion band Nuclear Martini playing on Luna. Nobody realized the hairy drummer was really a Hai in disguise."
 			"I saw a Hai wearing the cutest bonnet the other day. When I get back to human space, I'm definitely getting a pet squirrel."
 			"Do you see that Hai woman carrying her baby? Doesn't it look like a long-tailed teddy-bear? The Hai are so cute."
+			"Do you ever wonder what the Hai would think if they saw an actual squirrel? Do they even know that squirrels exist? Are squirrels just miniature versions of the Hai? I'm so lost."
+			"I always felt so rejected working as a servant on Shangri-la. Ever since I bought a ticket and ended up in Hai space, everyone around me has made me feel like I'm worth something."
+			"I bet if I bought a giant squirrel costume I could make my Hai friends think I was a complete stranger. They'd probably get a kick out of that."
+			"A traveling theater group passed through here a few days ago. They performed an adorable scene about a merchant captain who gives up his life and settles down with his family on Icelake."
+		word
+			`"`
+
+news "cute hai story in Hai space"
+	location
+		government "Hai"
+	name
+		word
+			"Hai civilian"
+			"Hai tour guide"
+	message
+		word
+			`"`
+		word
+			"Humans are always so curious when they discover our space past the wormhole. It is truly our pleasure to welcome them all and be amongst them."
+			"The little bit of hair that humans have on the tops of their head are so cute. I love how many ways they style it."
```

Comment:
The hair is the subject.
```suggestion
			"The little bit of hair that humans have on the top of their head is so cute. I love how many ways they style it."
```

Reply:
Yup, that needs to change. There's some strangeness with the second possible suggestion: I can't quite put my finger on it right now, but I think something needs to be changed. But I'd just go with the first of these two anyways; it feels more natural to me.